mesop/examples/async_await.py

```diff
@@ -30,4 +30,3 @@ async def click_async(e: me.ClickEvent):
   me.state(State).val1, me.state(State).val2 = await asyncio.gather(
```

Comment:
Please create a new function (and button) without doing the yield and keep the existing function and then update the e2e test to cover this case https://github.com/google/mesop/blob/fee3719827a1cd1ebfb419a639c1426279d69671/mesop/tests/e2e/async_await_test.ts#L4

For testing information, see: 
- https://google.github.io/mesop/internal/testing/#e2e-tests
- https://google.github.io/mesop/guides/testing/#running-and-debugging-a-test

Reply:
done